### DISCUSSION TITLE
Serving public static files

### DISCUSSION THREAD

**Discussion by gyl89** [necolas/react-native-web#2220] [Help]
I am trying to use universal links for my mobile/web app. For that, I need to serve two files  (https://docs.expo.dev/guides/linking/#universaldeep-links-without-a-custom-scheme) under a subpath` /.well-known`.

Is there a direct way to serve these files? My react native web app is deployed on vercel.

I have tried to use next for that purpose but I get the error https://github.com/vercel/next.js/issues/21703 while the solution there does not work. Thus, is there a simplier way to serve these files? Thanks

**Comment by sreuter** [comment]
Here you go: https://docs.expo.dev/guides/customizing-webpack/#editing-static-files

You can then just things in web/.well-known and the files will be added to the build.
